JoshuaOndieki/project-manager-app
# [Refactor] Restructure codebase to use Parcel and jest, and easy setup

Considering we're using modules, I have restructured our codebase to work in the browser by changing several things.

- Moved all JS, HTML, CSS, and assets into the src folder.
- Switched Vitest to jest
- Ignored all JSON files to avoid accidental commits of random test databases
- PARCEL. We're now able to bundle our project.

- `npm install` to install dependencies.
- Use `npm run dev` to start up a development server ready with a db already started.
- Use `npm run build` to bundle the app one time.
- Use `npm run tests` to run tests and see coverage.
Please comment on this PR if there's anything confusing with the new structure. Everything is as it was; just moved around.
@Gamalie @FLAMES39 

Comments are welcome even when PR is already merged!
